platform/darwin/src/MGLBaseStyleLayer.h

```diff
@@ -0,0 +1,26 @@
+#import <Foundation/Foundation.h>
+
+@interface MGLBaseStyleLayer : NSObject
+
+@property (nonatomic, assign, getter=isVisible) BOOL visible;
+
+/**
+ The maximum zoom level on which the layer gets parsed and appears on.
+ */
+@property (nonatomic, assign) float maxZoom;
+
+/**
+ The minimum zoom level on which the layer gets parsed and appears on.
+ */
+@property (nonatomic, assign) float minZoom;
+
+/**
+ Updates the layers layout and paint properties.
+ 
+ @discussion The properties are recalculated upon managing sources and
```

Comment:
The `@discussion` keyword is also unnecessary. Anything after the first blank line is treated as discussion by both Quick Help and jazzy.